spec/models/event_schedule_spec.rb

```diff
@@ -17,5 +17,47 @@
     it { is_expected.to validate_presence_of(:event) }
     it { is_expected.to validate_presence_of(:room) }
     it { is_expected.to validate_presence_of(:start_time) }
+
+    describe '#start_after_end_hour' do
+      context 'is invalid' do
+        it 'when event schedule start_time is after the conference end_hour, and returns an error message' do
+          conference = create(:conference)
+          event = create(:event, program: conference.program)
```

Comment:
create an `event_scheduled` instead an `event` and `event_schedule`: https://github.com/openSUSE/osem/blob/master/spec/factories/events.rb#L31